### ISSUE TITLE
Search over Preferences by the keyword "whitespace" doesn't find the tab "Grid" with the related option 

### ISSUE THREAD

**Issue by ask9** [dbeaver/dbeaver#35491]
### Description

https://github.com/user-attachments/assets/21a1f0ac-05ed-4f87-b96b-8fcece2f8fb4

### DBeaver Version

Community Edition 24.1.5-202408122322

### Operating System

Windows 10

### Database and driver

PostgreSQL 16
PostgreSQL JDBC Driver 42.7.4

**Comment by ask9** [comment]
By the way, I also recommend to add the keyword "hotkey" so that it finds a tab "Keys".
This is a commonly used term in many applications.

https://github.com/user-attachments/assets/7385bafc-1662-4e8d-a876-5f6c168ddb03

**Comment by E1izabeth** [comment]
Thank you for your request.

**Comment by jontrn** [comment]
Hello, I would like to work on this issue

**Comment by E1izabeth** [comment]
@jontrn OK

**Comment by ca1eigh** [comment]
Hi! Is this issue still available? I would like to work on it if possible. Thanks!

**Comment by ask9** [comment]
Hi @ca1eigh  As you can see, another user has already working on it.
If you want, you may choose some other issue clicking on the label "good first issue".

**Comment by jontrn** [comment]
Hi, just an update.

I'm struggling with figuring out where the search code would be located, I'm currently looking in org.jkiss.dbeaver.core. Would this be the correct location?

EDIT: Nevermind, I just found the PrefPage files, will look through those now! I think it'd be in the PrefPageQueryManager?

**Comment by E1izabeth** [comment]
@jontrn keywords for preference pages are defined in plugin.xml. Just search for tag `keyword`

**Comment by jontrn** [comment]
@E1izabeth Thanks! I've found the following, just unsure of where to go from here. I thought there might be a file with all the preferences settings that just had the white space option mislabeled.

![image](https://github.com/user-attachments/assets/b2bf0e23-a4d0-4a5e-a229-2f144598d37a)

EDIT: Nevermind, I figured it out :D

![image](https://github.com/user-attachments/assets/c22b67c4-613c-4398-a218-4404cb5e18ed)
